BOUNTY.md

```diff
@@ -0,0 +1,115 @@
+# GraphQL.NET Bounty Program
+
+The GraphQL.NET team has worked to create a program where community developers can work on specific new features
+of GraphQL.NET for compensation. These features are determined by GraphQL.NET maintainers to have the
+potential for high-impact improvement to the product. Design proposals that would fulfill a bounty may
+be made either by individuals or teams of developers, but the final payout will be made to only one stakeholder.
+
+
+## Bounty Process Flowchart
+
+1. The GraphQL.NET team publishes a Request For Proposals (RFP) describing a feature that the team has decided
+   should be prioritized, or a bug that is in need of being researched and fixed.
+    * Initial Bounty Status set to **Accepting Proposals**
+2. Developers write proposals for how they would implement the feature or fix the bug described in the RFP.
+3. The GraphQL.NET team works with developers to clarify and refine their proposals.
+4. A proposal is accepted. The developer(s) responsible for the proposal work to implement the feature or
+   bug fix according to their proposal.
+    * Bounty status changed to **In Progress**
+5. Pull requests are submitted to implement the proposal by the developers.
+    * Bounty status changed to **Under Review**
+6. Pull requests are reviewed by project maintainers, and the pull request developers make any changes to the
+   pull requests accordingly.
+7. Pull requests are merged into the master or develop branch.
+8. Developer(s) submit invoice to the [GraphQL.NET Open Collective](https://opencollective.com/graphql-net)
+   for bounty amount, confirmed by GraphQL.NET team members.
+    * Bounty status changed to **Claimed**
+9. Payment disbursed to developer(s) by the Open Source Collective from GraphQL.NET funds.
+
+
+## How to Claim a Bounty
+
+We've created a process to ensure multiple contributors aren’t competing on the same project, and to make sure
+work is properly merged into GraphQL.NET once completed.
+
+1. Read the Request For Proposal document completely.
+2. If you want to move forward, submit an proposal to the GraphQL.NET team on your suggested implementation by
+   replying to the RFP posting. Be sure to include as many details as requested by the spec, and be sure to address
+   each of the completion criteria. For proposals that include UI changes, mockups are strongly encouraged.
+3. Work with the GraphQL.NET team to finalize plans for the best implementation. Proposals will be judged on their
+   maintainability, design quality, and adherence to spec. Note that other developers may submit competing proposals
+   at this time as well.
+4. If your proposal is accepted, you may begin code implementation that adheres to your proposal.
+5. Submit code as a draft pull request in the appropriate repository so GraphQL.NET community leaders can track progress.
+6. Once code is complete, remove draft status and notify GraphQL.NET maintainers.
```

Comment:
_notify_ should be clarified

Reply:
I'm not sure how.  They just post in the PR just like all other communication.  Seems pretty obvious to me.

Want to add something more specifically like "request PR review from a maintainer"?

Reply:
Yes. Notify sounds not formal.

Reply:
```suggestion
6. Once code is complete, remove draft status and request a review from the GraphQL.NET maintainers.
```